openwisp_radius/tests/test_models.py

```diff
@@ -137,6 +121,28 @@ def test_convert_called_station_id(self):
                     radiusaccounting.called_station_id, 'CC-CC-CC-CC-CC-0C'
                 )
 
+    @capture_any_output()
+    @mock.patch.object(
+        app_settings,
+        'CALLED_STATION_IDS',
+        _CALLED_STATION_IDS,
+    )
+    @mock.patch.object(app_settings, 'OPENVPN_DATETIME_FORMAT', u'%Y-%m-%d %H:%M:%S')
+    @mock.patch.object(app_settings, 'CONVERT_CALLED_STATION_ON_CREATE', True)
+    def test_convert_called_station_id_with_organization_slug(self):
+        self.convert_called_station_id_tests()
+
+    @capture_any_output()
+    @mock.patch.object(app_settings, 'OPENVPN_DATETIME_FORMAT', u'%Y-%m-%d %H:%M:%S')
+    @mock.patch.object(app_settings, 'CONVERT_CALLED_STATION_ON_CREATE', True)
+    def test_convert_called_station_id_with_organization_id(self):
+        setattr(
+            app_settings,
+            'CALLED_STATION_IDS',
+            _CALLED_STATION_IDS,
+        )
+        self.convert_called_station_id_tests()
```

Comment:
how is this testing `organization_id` ?

Reply:
Thanks for pointing out, initially I added the line to replace org slug with org UUID, but while refactoring tests (creating `convert_called_station_id_tests` function), I think it got removed.